docs/software/media.md

```diff
@@ -843,4 +846,105 @@ See also
 - Official documentation: <https://komga.org/guides/>
 - Source code: <https://github.com/gotson/komga>
 
+## BruteFIR - Digital Room Correction via ALSA loopback in real time
+
+The DietPi installation of BruteFIR was created specifically for the GiBi project with Jean and Gilles. Details of which will be disclosed at a later date.
+
+BruteFIR applies Digital Room Correction (DRC) via ALSA loopback in real time. This will allow you to highly customize and tweak audio for a specific location, whilst having it applied to all audio played on your device.  
+For true audiophiles!
+
+Recommendations:
+
+- BruteFIR can be CPU intensive as it processes both L and R channels separately. A Raspberry Pi 2 or faster is highly recommended.
+- A HiFiBerry/external 24bit sound card for maximum audio quality.
+
+![DietPi media server software BruteFIR](../assets/images/dietpi-software-media-brutefir.gif){: style="width:500px"}
+
+=== "Configuration and Settings"
+
+    The configuration and settings are located there:
+
+    - `/etc/BruteFIR/null-65k48-l.pcm`: Null DRC filter (left channel)
+    - `/etc/BruteFIR/null-65k48-r.pcm`: Null DRC filter (right channel)
+    - `/etc/BruteFIR/highpass_l.pcm`: High pass DRC filter for testing (left channel)
+    - `/etc/BruteFIR/highpass_r.pcm`: High pass DRC filter for testing (right channel)
+    - `/etc/asound.conf`: Contains the BruteFIR ALSA loopback settings.
+    - `/var/log/brutefir`: Contains the BruteFIR log file.
+    - `/etc/BruteFIR/brutefir_config`: This file contains the EQ and DRC `.pcm` file location settings to be used.
+
+    If you make any changes, you will need to restart BruteFIR for the changes to take effect.
+
+=== "Restart BruteFIR service"
+
+    Restart the BruteFIR service with:
+
+    ```sh
+    dietpi-services restart
+    ```
+
+=== "HiFiBerry / External sound card"
+
+    If you are running a 24 bit external sound card, you can increase the sound quality by using `S32_LE` instead of `S16_LE`.
+
+    Change all `S16_LE` references to `S32_LE` in the following files:
+
+    - `/etc/asound.conf`
+    - `/etc/BruteFIR/brutefir_config`
+
+    Then restart BruteFIR.
+
+See also <https://torger.se/anders/brutefir.html>.
+
+## Raspotify - Spotify Connect client
+
+Spotify Connect client for the Raspberry Pi that Just Works™.
+
+!!! note "You will need a Spotify Premium account in order to use the Spotify Connect client."
+
+![DietPi media server software Raspotify](../assets/images/dietpi-software-media-raspotify.png){: style="width:250px"}
+
+Raspotify should work straight out of the box.
+
+=== "Changing options"
+
+    However, you can change the available options in:
+
+    - `/etc/default/raspotify`
+
+    Then restart services with:
+
+    ```sh
+    dietpi-services restart
```

Comment:
As always, I'm fans of restarting the single service only instead of all installed ones:
```
systemctl restart raspotify
```

Reply:
OK, changed.